include/tester/vtcp/tcp/TCPListener.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include<iostream>
+#include<functional>
+
+#include<boost/asio.hpp>
+#include<boost/bind/bind.hpp>
+
+using boost::asio::ip::tcp;
+
+class TCPListener
+{
+public:
+
+    TCPListener(int port, boost::asio::io_context & io_context,
+    std::function<void(const int, tcp::socket)> onClientConnected) :
```

Comment:
change the first parameter type of `onClientConnected` to `std::size_t` rather than `int`

Reply:
done